scripts/maint/release.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+"""Script to help release new versions."""
+import sys
+
+
+def main(args):
+    version = args[0]
+    print("# This script will guide you to create a new release, but it will "
+          "not do anything.")
```

Comment:
I don't understand why this is a script?
Can it just be a document with instructions?

Reply:
Because it should actually create the new release at some point.
For now just prints the correct versions, a bit more help that just instructions.